src/main/kotlin/deadline/application/DeadlineCreator.kt

```diff
@@ -17,9 +17,9 @@ class DeadlineCreator(private val repository: DeadlineRepository) {
      */
     fun create(primitives: DeadlinePrimitives) = create(Identifier(primitives.noteIdentifier), Time(primitives.time))
 
-    private fun create(noteId: Identifier, time: Time) = create(Deadline(noteId, time))
+    private fun create(noteId: Identifier, time: Time) = save(Deadline(noteId, time))
 
-    private fun create(deadline: Deadline): Deadline {
+    private fun save(deadline: Deadline): Deadline {
         repository.search(deadline.noteId)?.let { throw AlreadyConfiguredDeadline() }
         repository.create(deadline)
         return deadline
```

Comment:
I dont't get it, we use domain language for public methods but not for private methods?

Reply:
Maybe it would make sense that use case methods are called `create` and repository methods `save`. But not both at the same time on the use case.

Reply:
Maybe we should make a distinction on #43 about the naming on repositories.

If it the object we want to save it's already created, we should name it "save" 
If the object we want to create and save isn't created, we should call it "create"